start.sh

```diff
@@ -18,15 +18,101 @@ usage() {
     echo 'that command, for testing purposes.'
 }
 
+tritonConfigured() {
+
+    # is node-triton installed?
+    local triton_installed=0
+    command -v triton >/dev/null 2>&1 && triton_installed=1
+    if [ "0" = "$triton_installed" ]; then
```

Comment:
You're right to call me out for being less efficient (and readable) in my version.